elfo-core/src/request_table.rs

```diff
@@ -303,6 +303,11 @@ impl<R> ResponseToken<R> {
     pub fn is_forgotten(&self) -> bool {
         self.data.is_none()
     }
+
+    #[cfg(any(test, feature = "test-util"))]
+    pub fn test() -> Self {
+        ResponseToken::forgotten()
+    }
```

Comment:
Didn't find a way to create `ResponseToken` in test without this change.

Reply:
mark `forgotten()` as `#[doc(hidden)] #[inline]` and use it for tests inside elfo.

the test-util feature is supposed to be used in user code, where response tokens should never be possible to create